### PR TITLE
Add guzzle options to scraper

### PR DESCRIPTION
**Pull request by lmfmaier**
I have added a constructor to Scraper which allows passing options for the Guzzle http client. There are two default options (connect_timeout and User-Agent)

The scraper can now be instantiated with options for guzzle:

`$scraper = new Scraper(
                ['headers' =>
                    ['User-Agent' => 'facebookexternalhit/1.1']
                ]
            );`

### CODE REVIEW

File: src/Parser/OgParser.php [hunk 1]
```diff
@@ -24,64 +24,64 @@ public function parse(string $content): Meta
             $meta->setTitle(htmlspecialchars_decode($matches[1]));
         }
 
-        preg_match('/<meta.*name=\"description\".*content=\"(.+)\"\s*[\/]*\>/Uis', $content, $matches);
+        preg_match('/<meta.*name=\"description\".*content=\"(.+)\"/Uis', $content, $matches);
```
Comment (tomaj) [inline src/Parser/OgParser.php:27]: why we need this changes?

Comment (lmfmaier) [reply src/Parser/OgParser.php:27]: The pattern is too greedy, it grabs additional (probably non standard) attributes after the "content" attribute, for example I encountered the following code:

`<meta name="apple-itunes-app" content="app-id=1282107098" class="next-head" />`

The regex grabs `class="next-head"` also

Comment (lmfmaier) [reply src/Parser/OgParser.php:27]: This solution just skips everthing after the content-attribute, there may be better solutions, I have another look

Comment (tomaj) [reply src/Parser/OgParser.php:27]: looks good, can you please add test for it?

Comment (lmfmaier) [reply src/Parser/OgParser.php:27]: I have added a test for non standard attributes after the "content" attribute, all tests including the original ones work

**Pull request merged**